app/api/answers/get-answers-with-attempt-id.feature

```diff
@@ -26,11 +26,11 @@ Background:
     | 200 | Category | false          | false    | 1234,2345      | true               | 0        |
     | 210 | Category | false          | false    | 1234,2345      | true               | 0        |
   And the database has the following table 'groups_items':
-    | ID | idGroup | idItem | sFullAccessDate | bCachedFullAccess | bCachedPartialAccess | bCachedGrayedAccess | idUserCreated | iVersion |
-    | 42 | 13      | 190    | null            | false             | false                | false               | 0             | 0        |
-    | 43 | 13      | 200    | null            | true              | true                 | true                | 0             | 0        |
-    | 44 | 13      | 210    | null            | false             | false                | true                | 0             | 0        |
-    | 45 | 41      | 200    | null            | true              | true                 | true                | 0             | 0        |
+    | ID | idGroup | idItem | sCachedFullAccessDate | sCachedPartialAccessDate | sCachedGrayedAccessDate | idUserCreated | iVersion |
+    | 42 | 13      | 190    | 3017-05-29T06:38:38Z  | 3017-05-29T06:38:38Z     | 3017-05-29T06:38:38Z    | 0             | 0        |
```

Comment:
3017? Not very domain-driven data ;-) It may even fail in some date systems.
I think 2030 is already enough in the future. :-)

Reply:
Fixed. It will be 2037. FYI:
```
mysql> select unix_timestamp('2037-12-31T23:59:59');
+---------------------------------------+
| unix_timestamp('2037-12-31T23:59:59') |
+---------------------------------------+
|                            2145916799 |
+---------------------------------------+
1 row in set (0.00 sec)

mysql> select unix_timestamp('2038-01-19');
+------------------------------+
| unix_timestamp('2038-01-19') |
+------------------------------+
|                   2147472000 |
+------------------------------+
1 row in set (0.00 sec)

mysql> select unix_timestamp('2038-01-20');
+------------------------------+
| unix_timestamp('2038-01-20') |
+------------------------------+
|                            0 |
+------------------------------+
1 row in set (0.01 sec)
```